app/src/main/java/mozilla/lockbox/view/ItemListFragment.kt

```diff
@@ -205,7 +210,7 @@ class ItemListFragment : CommonFragment(), ItemListView {
     }
 
     override fun loading(isLoading: Boolean) {
-        if (isLoading) {
+        if (isLoading && !isRefreshing) {
```

Comment:
We're loading more tricky-to-unit-test logic into the `View` here, which makes me nervous despite your slick new ui test :) What would you think about making the `isRefreshing` variable a part of the `ItemListView` interface and incorporating that in the `ItemListPresenter`? The `ItemListPresenter` can then decide whether to push the `loading` update or not.